src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncTask.java

```diff
@@ -27,7 +27,7 @@
  * It will clean up leases/activities for shards that have been completely processed (if
  * cleanupLeasesUponShardCompletion is true).
  */
-class ShardSyncTask implements ITask {
+class ShardSyncTask implements ITask, Runnable {
```

Comment:
`ITask` is already a `Callable`, you don't need a `Runnable` to schedule it to a thread.

Reply:
Yes it was redundant (leftover from previous commit on our fork). Removed.